components/eamxx/tests/coupled/dynamics_physics/homme_shoc_cld_spa_p3_rrtmgp_dp/input.yaml

```diff
@@ -22,6 +22,10 @@ initial_conditions:
   surf_sens_flux: 0.0
   precip_liq_surf_mass: 0.0
   precip_ice_surf_mass: 0.0
+  perturbed_fields: [T_mid]
+  perturbation_limit: 0.001
+  perturbation_pressure_lower_bound: 900.0 # in milibar
+  perturbation_random_seed: 1805289383
```

Comment:
@PeterCaldwell Do you see needed a different perturbation range (defined by `perturbation_limit`) for each field? Currently I just have it set to a single value used for every field.

Reply:
We probably do need different perturbations for different variables because T has values of ~300 K while nc has units of something like 1e5 #/kg. Maybe just a different scaling factor for each variable?

Reply:
This value isn't an absolute but I guess is like a "percentage perturbed". So it will scale with variable magnitude. Basically `pert_val` is in `[-pertlim, pertlim]` and then `f_val_new = f_val*(1+pertval)`. Maybe this could be better described in the name..

Reply:
>This value isn't an absolute but I guess is like a "percentage perturbed"
Ah, percentage perturbed is perfect. I think it is very reasonable to just use the same percentage perturbation for all variables. Much better than asking the user to specify a zillion different scaling factors.